### ISSUE TITLE
Search fields lack visual and programmatically associated labels [Android]

### ISSUE THREAD

**Issue by jonnyjames** [covid-alert-ny/covid-green-app#303]
The “search” text field found in the “Select your county” overlays relies on the “Search” placeholder text for a visual label. Once a user has entered data in the field, the “search” placeholder text is gone, the field is still announced as whatever data is in the field, followed by the search label, but no visible label is present. The overlay's "Choose..." text doesn't assist in describing the purpose of the overlay's search input.

![Search - visible label](https://user-images.githubusercontent.com/7034772/92136433-045bcd00-edda-11ea-8ec0-0faf077df5f4.png)

**Remediation Guidance**

We suggest adding a text label that remains visible at all times, then programmatically associate the label by including it in the `accessibilityLabel` attribute on the text field. Hide the visible text label from TalkBack users via the `accessibilityElementsHidden` attribute, as not to announce the label twice to said users.

```javascript
<View accessibilityElementsHidden={true}>
  <Text>Search</Text>
</View>
<TextInput accessibilityLabel='Search' />
```
As an alternative to the visual text label, a “search” icon (e.g. magnifying glass icon) could be added to the beginning or end of the text field. The `accessibilityLabel` and `accessibilityElementsHidden` guidance from above would still apply.

**Comment by floridemai** [comment]
@jonnyjames This should be addressed in the next build. Not that the  search label doesn't update right after you make edits, but the labels and roles should be ok.

**Comment by jonnyjames** [comment]
@floridemai The search field still lacks a visible "Search" text label once a user enters their search query. We recommended a static, visible label, or a using an icon to convey its purpose (magnifying glass icon). Let me know if you require further help with this issue.

**Comment by helloantoine** [comment]
Worked with @ckiss to add a search icon always visible.

**Comment by floridemai** [comment]
@jonnyjames https://github.com/project-vagabond/covid-green-app/issues/239#issuecomment-689348332
Any input would be highly appreciated.

**Comment by ckiss** [comment]
fixed, avail in next build

**Comment by jonnyjames** [comment]
@floridemai @ckiss I am not seeing the magnifying glass icon in the Android build.  Please advise.

**Comment by jonnyjames** [comment]
@floridemai @ckiss I am seeing it now after the update. Closing this issue.
